src/SOS/Strike/util.cpp

```diff
@@ -3209,13 +3277,18 @@ void ReloadSymbolWithLineInfo()
         if (!(Options & SYMOPT_LOAD_LINES))
         {
             g_ExtSymbols->AddSymbolOptions(SYMOPT_LOAD_LINES);
-            
-            if (SUCCEEDED(g_ExtSymbols->GetModuleByModuleName(MSCOREE_SHIM_A, 0, NULL, NULL)))
-                g_ExtSymbols->Reload("/f " MSCOREE_SHIM_A);
-            
+            if (SUCCEEDED(g_ExtSymbols->GetModuleByModuleName("mscoree.dll", 0, NULL, NULL)))
```

Comment:
Any reason here we don't use a defined constant as we do for the others?

Reply:
I put these desktop specific defines back and reverted these changes.